hack/boilerplate.go.txt

```diff
@@ -0,0 +1,15 @@
+/*
```

Comment:
Is this required for this change?

Reply:
it is if I want the generator be as close to a modern generation as possible yes? Like this was basically "make the new generation method work on the old stuff with as few changes as possible".